src/invidious/helpers/utils.cr

```diff
@@ -389,3 +389,56 @@ def reduce_uri(uri : URI | String, max_length : Int32 = 50, suffix : String = "
   end
   return str
 end
+
+# Get the html link from a NavigationEndpoint or an innertubeCommand
+def parse_link_endpont(endpoint : JSON::Any, text : String, video_id : String)
```

Comment:
You meant `parse_link_endpoint` (with a `i`)? :P

Reply:
Updated :-)